lib/rubycritic/cli/options.rb

```diff
@@ -19,8 +19,14 @@ def parse
         self
       end
 
+      # :reek:NilCheck
```

Comment:
Does this disable the NilCheck for the method, to the end of file, or "from here on" ?

Reply:
https://github.com/troessner/reek/blob/master/docs/Smell-Suppression.md